src/components/crag/cragSummary.tsx

```diff
@@ -117,7 +117,7 @@ export default function CragSummary ({ area, history }: CragSummaryProps): JSX.E
    * Initially set to the childAreas prop coming from Next build, the cache
    * may be updated by the users in the AreaCRUD component.
    */
-  const [childAreasCache, setChildAreasCache] = useState(childAreas)
+  const [childAreasCache, setChildAreasCache] = useState(sortByLeftRightIndex(childAreas))
```

Comment:
One question worth discussing is whether we should be sorting in the frontend or backend. You could argue that frontend is better if we want to allow dynamic sorting (L->R, Alphabetical, Most children) one day, so the backend result remain invariant and can be cached. Backend sorting just simplifies frontend code a bunch especially in the case where we always sort L->R. 

I realize that for areas, L->R sorting doesn't make too much sense because areas aren't lined up on a wall like climbs. So the main purpose of the sorting here is just a stable ordering. If stability is what we care about, then perhaps no point enabling multiple sort orders for areas, which would lead to backend sorting being better.

For now I'm just following the design pattern of climbs, and doing sorting on the frontend.

Reply:
All good points.  Agreed we should sort in the backend.  Can you create a backend issue?

Reply:
https://github.com/OpenBeta/openbeta-graphql/issues/294 Sounds good. I can work on this next.